contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -179,6 +180,839 @@ const char *bbf_servicename = "MSSQLSERVER";
 char	   *bbf_language = "us_english";
 #define MD5_HASH_LEN 32
 
+
+static const struct
+{
+	const char *stdname;		/* Windows name of standard timezone */
+	const char *dstname;		/* Windows name of daylight timezone */
+	const char *pgtzname;		/* Name of pgsql timezone to map to */
+}			win32_tzmap[] =
+
+{
+	/*
+	 * This list was built from the contents of the registry at
+	 * HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Time
+	 * Zones on Windows 7, Windows 10, and Windows Server 2019.  Some recent
+	 * additions have been made by comparing to the CLDR project's
+	 * windowsZones.xml file.
+	 *
+	 * The zones have been matched to IANA timezones based on CLDR's mapping
+	 * for "territory 001".
+	 */
+	{
```

Comment:
as we are relying on postgres for core conversion, how are we making sure that every entry in the list works as expected for SQL server

Reply:
Here in the mapping it is said that the list in taken from registry of windows which is the same directory as mentioned in tsql doc , so the mapping here are correct.
```
This list was built from the contents of the registry at HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Time Zones on Windows 7, Windows 10, and Windows Server 2019.  Some recent additions have been made by comparing to the CLDR project's windowsZones.xml file.
```

Reply:
but that does not guarantee that for each entry BBF will give the same result as TSQL, have we tested these? Do we plan to add test for each entry?

Reply:
Will add test case for each timezone .